### DISCUSSION TITLE
Warn on usages of hidden fields in form definitions

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5669] [Feature Requests]
For a form editor when we eventually have it:
If form definition uses fields that are marked as hidden, there should be warnings emitted
This is because such fields won't be easily visible in the query builder or work bench

**Comment by grantfitzsimmons** [comment]
:+1: Would be really useful!

**Comment by maxpatiiuk** [comment]
Probably should be part of the form editor v2
